### PR TITLE
Adds types for @gburnett/draft-js

### PR DESCRIPTION
**Pull request by gburnett**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [ ] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [ ] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by gburnett** [comment]
Deviations, mentioned above, follow the pattern found in [types/draft-js](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/draft-js)

File: types/gburnett__draft-js/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "rules": {
+        "dt-header": "off"
```
Comment (gabritto) [inline types/gburnett__draft-js/.eslintrc.json:3]: Why do you need to turn this rule off?

File: types/gburnett__draft-js/tslint.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (gabritto) [inline types/gburnett__draft-js/tslint.json:1]: Why do you need to turn those rules off? It's preferred to follow the standard style on DT, or add inline annotations to the places where a rule must be broken.

File: types/gburnett__draft-js/tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": false,
```
Comment (gabritto) [inline types/gburnett__draft-js/tsconfig.json:10]: Could you make the types work with `strictNullChecks` on?

**Review by gabritto — CHANGES_REQUESTED** [review]
Please either use the standard DT settings, or explain why changing them are necessary. Thanks.

**Pull request closed without merging**